get_weather_data.ipynb

```diff
@@ -0,0 +1,98 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "963fbd82",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import meteostat as ms\n",
+    "from datetime import datetime\n",
+    "from string import punctuation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bd1c821b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_weather_data(place,start,end=datetime.now):\n",
```

Comment:
In the future I think we should move these function/ class definitions to a `almanac` package of .py files that is then imported into any notebooks we're working with